packages/components/pages/f-checkout/src/services/addressService/tenants/es.js

```diff
@@ -0,0 +1,29 @@
+function isFullPostCode (postcode) {
+    return !postcode ? false : postcode.length === 5;
```

Comment:
Inverting the logic might make it a little more readable?
```suggestion
    return postcode && postcode.length === 5;
```
or simply
```suggestion
    return postcode?.length === 5;
```
Although this would succeed if you pass in an array of length 5 😄

Reply:
Thanks